src/index.js

```diff
@@ -234,18 +235,20 @@ const injectLyrics = (lyrics, wrapper) => {
       let target_language = items.translationLanguage || "en"; // Use the saved language or the default 'en'
 
       if (item.words.trim() !== "♪" && item.words.trim() !== "") {
-        translateText(item.words, target_language).then((translatedText) => {
-          if (translatedText) {
-            // If the translation was successful, set the translated text as the content for translatedLine
-            translatedLine.textContent = "\n" + translatedText;
+        translateText(item.words, target_language).then((result) => {
+          if (result) {
+            if (result.originalLanguage !== target_language) {
```

Comment:
```suggestion
          if (result && result.originalLanguage !== target_language) {
```

Reply:
this causes a translation error to be thrown
![image](https://github.com/boidushya/better-lyrics/assets/80891624/0c69fdaa-2b36-48c1-b411-024d4c70c545)

Reply:
Ah makes sense! all good